src/components/StepBar/StepBar.module.scss

```diff
@@ -1,48 +1,51 @@
 @import '../../styles/typography';
 @import '../../styles/animations';
 @import '../../styles/colors';
+@import '../../styles/theme';
 
 .StepBar {
   position: relative;
   width: 100%;
   font-size: #{$size-tiny};
-  font-weight: #{$font-weight-bold};
   text-align: center;
 
+  display: flex;
+  justify-content: space-around;
+
+  user-select: none;
+
   &::before {
     content: ' ';
     position: absolute;
     bottom: 0;
     left: 0;
     width: 100%;
-    border-bottom: 1px solid #{$mine-shaft};
+    border-bottom: 1px solid $color-primary-4;
   }
 
   .Step {
-    cursor: pointer;
     transition: opacity #{$time-long} ease-in-out;
     z-index: 2;
+    flex: 1;
+    padding: 0.5rem 2rem;
+
+    color: $color-greyscale-11;
+
+    &[data-selected] {
+      color: $color-greyscale-12;
+    }
 
-    > a {
-      color: #{$blue};
-      transition: text-shadow 0.1s ease-in-out;
+    // if its not data-selected or data-disabled, then it's clickable
+    &:not([data-selected]):not([data-disabled]) {
+      cursor: pointer;
     }
   }
 
   .Bar {
-    padding-bottom: 4px;
-    color: #{$blue};
-    border-bottom: 1px solid #{$blue};
-    transition: all #{$time-long} ease-in-out;
-
-    &::after {
-      content: attr(data-text);
-    }
+    position: absolute;
+    top: calc(100% - 1px);
 
-    &.Hide {
-      border-bottom: 1px solid #{$black};
-      color: #{$black};
-      z-index: -1;
-    }
+    border-bottom: 1px solid $color-primary-9;
+    transition: all 0.2s ease-in-out;
```

Comment:
nit: I think there may be a `0.2s` in `_animations.scss` we could import here.

Reply:
May leave it as is now until we sort animation tokens